src/main/java/com/powsybl/ws/commons/computation/service/AbstractWorkerService.java

```diff
@@ -116,16 +116,25 @@ protected boolean resultCanBeSaved(R result) {
         return result != null;
     }
 
+    protected void addLogsWhenFailed(C runContext, AtomicReference<ReportNode> rootReportNode) {
```

Comment:
A better name might be: addLogs() ? Because there will be no check in this method for the "whenFailed"? Or will be?